tpchgen/src/generators.rs

```diff
@@ -1404,6 +1404,27 @@ impl fmt::Display for ClerkName {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd)]
```

Comment:
I also changed this from `char` to an enum so I could more easily convert it to StringViewArray 

I actually think I can apply the same idea for some of the status codes in LineItem and make it faster too. As a follow on PR

Reply:
Follow up the idea is here: https://github.com/clflushopt/tpchgen-rs/pull/75 (but still not complete)